cql3/expr/expression.cc

```diff
@@ -1483,6 +1544,10 @@ std::ostream& operator<<(std::ostream& s, oper_t op) {
         return s << "=";
     case oper_t::NEQ:
         return s << "!=";
+    case oper_t::EQ_SPECIAL_NULL:
+        return s << "EQUALS_SPECIAL_NULL";
+    case oper_t::NEQ_SPECIAL_NULL:
+        return s << "NOT_EQUALS_SPECIAL_NULL";
```

Comment:
It's better to use `=` and `!=` to avoid surprising the users, even if we lost a little bit of information here.

Reply:
I agree. If the users used "=" in an expression, they would want to see "=" in the error message, not "EQUALS_SPECIAL_NULL" or "=*".
By the way, now that I think of it, I have some doubts if we really need the EQ_SPECIAL_NULL operator. Instead of being a different operator, it could have been an option to evaluate() on how to evaluate the regual EQ operator.

Reply:
I agree that a flag in `evaluation_inputs` that specifies in which context we are evaluating would be cleaner
I think the problem is that we would have to remember to set this flag in all the right places. With this solution we just generate the right data in the parser and then we don't have to worry that we might have missed setting the flag somewhere.
Another option would be having the flag inside `binary_operator`, something like `is_special_lwt_null_binop`

Reply:
Changed to have a separate enum in binary_operator.

Printing now shows plain `=` in user mode, but annotated in debug mode.